crates/generated_parser/src/ast_builder.rs

```diff
@@ -56,6 +56,9 @@ enum BindingKind {
 
     // BindingIdentifier is the name of ClassDeclaration.
     Class,
+
+    // BindingIdentifier is the name of Label.
+    Label,
```

Comment:
this is a bit hacky, but it is also what has me thinking that this should all live together

Reply:
Now that i think of it, it would be better to do this the same way as `is_labelled_function`, I will take care of it tomorrow